test/karma/karma.conf.js

```diff
@@ -72,6 +72,7 @@ module.exports = function( config ) {
 
 		colors: !isTravis,
 
+		hostname: "bs-local.com",
```

Comment:
This does not look right to me, and does not work in my local testing.

Reply:
@gibson042 What doesn't work here? `bs-local.com` is a public domain that resolves to `127.0.0.1`. It's required because some iOS versions don't allow to remap localhost.

Granted, it introduces the need to have internet connection for local testing. But you can add this domain to `/etc/hosts` so that it's not necessary.